network-api/networkapi/mozfest/templates/partials/primary_mozfest_nav.html

```diff
@@ -39,7 +39,9 @@
                   </div>
                 </div>
               </div>
+              {% if menu_root.specific.cta_button_destination and menu_root.specific.cta_button_label %}
```

Comment:
this might be better to turn into a fragment then, so we can say `{% include "fragments/primary_cta.html" with root=menu_root.specific %}` and then we put these three lines in that fragment so that it does nothing if there's no destination/label

Reply:
Love this suggestion!

Reply:
Done!